options/linux/include/sys/epoll.h

```diff
@@ -3,8 +3,8 @@
 
 #include <stdint.h>
 #include <abi-bits/signal.h>
+#include <abi-bits/epoll.h>
 
-#define EPOLL_CLOEXEC 1
 #define EPOLL_NONBLOCK 2
```

Comment:
Shouldn't we also move `EPOLL_NONBLOCK` at the same time?

Reply:
I'm unsure, I don't see `EPOLL_NONBLOCK` on my host so I can't tell if the value is correct.

Reply:
Seems to be a glibc artifact: https://sourceware.org/bugzilla/show_bug.cgi?id=14985
We should probably remove it entirely.

Reply:
Will do